demisto_sdk/commands/common/errors.py

```diff
@@ -959,7 +959,9 @@ def invalid_script_id(script_entry_to_check, pb_task):
     def content_entity_version_not_match_playbook_version(main_playbook, entities_names, main_playbook_version):
         return f"Playbook {main_playbook} with version {main_playbook_version} uses {entities_names} " \
                f"with a version that does not match the main playbook version. The from version of" \
-               f" {entities_names} should be at least {main_playbook_version}."
+               f" {entities_names} should be at least {main_playbook_version}.\n" \
+               f"Possible reason for such an error, would be that the name of the sub-Pb in the parent Pb's yml file" \
```

Comment:
better to separate this option into another error.
when validating a playbook we search each sub-playbook in the ID set.
if we don't find it there - raise an error (no such playbook).
if we do find it, check the version and raise this error if needed (the logic we have now)